Snowflake.Data.Tests/UnitTests/ConnectionPoolManagerTest.cs

```diff
@@ -168,38 +168,25 @@ public void TestSetPoolingForAllPoolsDisabled()
 
             // Act
             var thrown = Assert.Throws<Exception>(() => _connectionPoolManager.SetPooling(false));
-            
+
             // Assert
             Assert.That(thrown.Message, Does.Contain("You cannot change connection pool parameters for all the pools. Instead you can change it on a particular pool"));
         }
 
         [Test]
-        public void TestGetPoolingOnManagerLevelWhenNotAllPoolsEqual()
+        public void TestGetPoolingOnManagerLevelAlwaysTrue()
         {
             // Arrange
             var sessionPool1 = _connectionPoolManager.GetPool(ConnectionString1, _password);
             var sessionPool2 = _connectionPoolManager.GetPool(ConnectionString2, _password);
             sessionPool1.SetPooling(true);
             sessionPool2.SetPooling(false);
-            
-            // Act/Assert
-            var exception = Assert.Throws<SnowflakeDbException>(() => _connectionPoolManager.GetPooling());
-            Assert.IsNotNull(exception);
-            Assert.AreEqual(SFError.INCONSISTENT_RESULT_ERROR.GetAttribute<SFErrorAttr>().errorCode, exception.ErrorCode);
-            Assert.IsTrue(exception.Message.Contains("Multiple pools have different Pooling values"));
-        }
 
-        [Test]
-        public void TestGetPoolingOnManagerLevelWorksWhenAllPoolsEqual()
-        {
-            // Arrange
-            var sessionPool1 = _connectionPoolManager.GetPool(ConnectionString1, _password);
-            var sessionPool2 = _connectionPoolManager.GetPool(ConnectionString2, _password);
-            sessionPool1.SetPooling(true);
-            sessionPool2.SetPooling(true);
-            
-            // Act/Assert
-            Assert.AreEqual(true,_connectionPoolManager.GetPooling());
+            // Act
+            var pooling = _connectionPoolManager.GetPooling();
+
+            // Assert
+            Assert.IsTrue(pooling);
```

Comment:
check my comment on the ConnectionPoolManager, imho it's misleading. Imagine a developer logging pool enabled flag using this interface - I would rather throw an error than return true in such a case

Reply:
We have several places in code when we call `SnowflakeDbConnectionPool.GetPooling()` for example in `SnowflakeDbConnection.CanReuseSession`. So if we would do such a change everything is getting more complicated. From the other side, returning always true for new pool makes sense - because generally pooling is always enabled, you can disable it only in particular pools by connection string parameter.

Reply:
Right, now you need to take i-th session pool and call GetPooling on it